### ISSUE TITLE
Typos and improvements

### ISSUE THREAD

**Issue by fediazgon** [shervinea/cheatsheet-translation#28]
Hi. I think in this issue we can discuss explanations that could be improved or typos that we have found. For example. while I was translating the Deep Learning cheatsheet to Spanish, I came across a few things:

* [IMPROVEMENT] in [21], instead of:

> ... F the size of the convolutional layer neurons

I would say " ... F the size of the convolution kernel". This is the word that is most commonly used in deep learning frameworks like [Keras](https://keras.io/layers/convolutional/).

* [IMPROVEMENT] also in [21], instead of:

> ... the number of neurons N that fit in a given volume is such that

I would say something like "... the size N of the output volume, is".

* [TYPO] in [22]:

> Batch normalization ― It is a step of hyperparameter γ,β that normalizes the batch {xi}.

Do you mean "It is a set of hyperparameters ... "?
